source/platform/native.rst

```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
+As of this writing, Velocity 1.1.0+, Sponge 8+, Paper 1.16.5+, and Minestom support Adventure
```

Comment:
It might be better to turn this into a table at this point - "Platform Name" and "Minimum Version" columns.